nl/nl_linux.go

```diff
@@ -789,3 +789,36 @@ func (sh *SocketHandle) Close() {
 		sh.Socket.Close()
 	}
 }
+
+// IPProto represents Flower ip_proto attribute
```

Comment:
im not sure if these bits should be here as they are specific to tc filter.

maybe move to `tc_linux.go` ?